cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/aggregator/KafkaPartitionMetricSampleAggregatorTest.java

```diff
@@ -109,15 +109,15 @@ public void testAggregateWithUpdatedCluster() throws NotEnoughValidWindowsExcept
 
     populateSampleAggregator(NUM_WINDOWS + 1, MIN_SAMPLES_PER_WINDOW, metricSampleAggregator);
 
-    TopicPartition tp1 = new TopicPartition(TOPIC, 1);
+    TopicPartition tp1 = new TopicPartition(TOPIC + "1", 0);
```

Comment:
Why did we change the topic and partition names?

Reply:
The scenario here is that there is a partition "topic-0" which has enough samples  and another partition("topic-1"->"topic1-0") which has none. 

If we do not change the topic name of second partition, after the code change 
https://github.com/linkedin/cruise-control/blob/6f133ece636dd8228b9f1895e73ceedbb67e6a91/cruise-control-core/src/main/java/com/linkedin/cruisecontrol/monitor/sampling/aggregator/WindowState.java#L70
we will get a `NotEnoughValidWindowsException`. 

This is because if tow partitions has the same  topic name("topic") the invalidity of one partition will invalid the other one(the aggregating granularity of `PartitionEntity` is `ENTITY_GROUP`). Before the code change, this is OK, given that a window with zero valid partition is taken as a valid window, so there will be NUM_WINDOWS valid with zero valid partition sample in them. But after the code change, all these window become invalid.